common/src/main/java/com/wynntils/features/ui/WynntilsContentBookFeature.java

```diff
@@ -83,28 +83,28 @@ public class WynntilsQuestBookFeature extends Feature {
             () -> WynntilsMenuScreenBase.openBook(WynntilsGuidesListScreen.create()));
 
     @RegisterConfig
-    public final Config<Boolean> replaceWynncraftQuestBook = new Config<>(true);
+    public final Config<Boolean> replaceWynncraftContentBook = new Config<>(true);
```

Comment:
No config upfixers?

Reply:
This was made in a rush to support 2.0.3. Also perhaps it is better to reset the configs this case..

Reply:
Okay